CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 **[ClickHouse release v22.3-lts, 2022-03-17](#223)**<br>
 **[ClickHouse release v22.2, 2022-02-17](#222)**<br>
 **[ClickHouse release v22.1, 2022-01-18](#221)**<br>
-**[Changelog for 2021](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/whats-new/changelog/2021.md)**<br>
+**[Changelog for 2021](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/reference/whats-new/changelog/2021.md)**<br>
```

Comment:
```suggestion
**[Changelog for 2021](https://clickhouse.com/docs/en/whats-new/changelog/2021/)**<br>
```

Reply:
When in the docs, we link to the markdown files that are also in the docs, and when Docusaurus generates the HTML it replaces the links to the markdown files with links to the HTML.  It gets easier to grok over time :)  I think the suggestion above will fix it.

Reply:
When in the docs, we link to the markdown files that are also in the docs, and when Docusaurus generates the HTML it replaces the links to the markdown files with links to the HTML.  It gets easier to grok over time :)  I think the suggestion above will fix it.